.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,87 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ 'jetty-10.[1-9]?[0-9].x', 'jetty-11.[1-9]?[0-9].x', 'jetty-12.[1-9]?[0-9].x' ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ 'jetty-10.[1-9]?[0-9].x', 'jetty-11.[1-9]?[0-9].x', 'jetty-12.[1-9]?[0-9].x' ]
```

Comment:
not sure we want to run this on PRs.
just a weekly on hardcoded branches might be enough.

Reply:
I'm ok with PRs, it will catch accidentally introduced bugs.